library_version_analysis.gemspec

```diff
@@ -20,8 +20,8 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'google-api-client'
   spec.add_dependency "googleauth"
-  spec.add_dependency "graphql", "~> 2.0.24"
-  spec.add_dependency "graphql-client", "~> 0.18"
+  spec.add_dependency "graphql", "~> 2"
+  spec.add_dependency "graphql-client", "~> 0.2"
```

Comment:
@daiaventureira This will actually need to be:

```
  spec.add_dependency "graphql-client", "~> 0.20"
```

Since there's a difference in semantic versioning betwween `0.2` and 0.20`. Sorry I might have suggested `0.2`.